README.md

```diff
@@ -46,6 +46,13 @@ $ source .venv/bin/activate
 $ make setup
 ```
 
+4. Running docker locally inside your shell 
+
+```bash
+$ sh run-local.sh
+
+```
+
```

Comment:
There's a section on README for running the script. 
Reference: 
https://github.com/leapfrogtechnology/oss-leaderboard#building

We can add more description here.

Reply:
Yup